packages/rolldown/tests/plugin/plugin.test.ts

```diff
@@ -0,0 +1,33 @@
+import { expect, test, vi } from 'vitest'
+import { rolldown, Plugin } from 'rolldown'
+
+async function buildWithPlugin(plugin: Plugin) {
```

Comment:
I think `fixture` based testing is enough to do the similar things, but let's evolve a little bit and see how this goes.

Reply:
Yeah. Here separate it because the `error` hook needs to catch the error, i want to avoid increasing fixture test complex.